lib/index.js

```diff
@@ -109,6 +109,9 @@ const createHandler = async function createHandler(options) {
     logger.verbose(`with ${inspect(config)}`);
   }
   logger.info(`  (use "bn deploy${optPath}${optName}" to deploy the function)`);
+  if (experimentalRuntimes.includes(options.runtime)) {
+    logger.warn(`Please be aware, support for ${options.runtime} is currently experimental`);
```

Comment:
I would go for a shorter version, such as ` . [notice: java8 is experimental]` (also notice the two spaces).
That said, better have @avnerbraverman or @urirb take a look at the message.